feat: improve docs for agones dependency (#7)
GameFabric/gswrapper: README.md

The game server wrapper, also known as `gswrapper` or `gsw`, can be used to run a game server executable
from within the container image that runs your GameFabric Armadas or Vessels.

> The wrapper is dependency-free with one exception.
> It communicates with the Agones sidecar container via localhost which is always available in GameFabric.
> If you want to run it locally for development, you need run the Agones SDK server dummy locally
> (see [Agones documentation](https://agones.dev/site/docs/guides/client-sdks/local/)).

Reviewer: ```suggestion
The GSW is dependency-free with one exception.
It communicates with the Agones sidecar container via localhost which is always available in GameFabric.
If you want to run it locally for development, you need run the Agones SDK server dummy locally
(see [Agones documentation](https://agones.dev/site/docs/guides/client-sdks/local/)).
```

I tried to be consistent with how we call _it_ throughout this README.

> 
> 

Reviewer: ```suggestion

```

> for more documentatiion please use the [docs](https://docs.gamefabric.com/multiplayer-servers/multiplayer-services/game-server-wrapper)

Reviewer: ```suggestion
For further documentation see our [GameFabric docs of the GSW](https://docs.gamefabric.com/multiplayer-servers/multiplayer-services/game-server-wrapper).
```

WDYT?

Would move this sentence to below the list of features (L22) as the "ending phrase" of the introduction.
